src/v2i-hub/SampleBSMPlugin/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name":"SampleBSMPlugin",
+	"description":"Sample BSMplugin for making websocket connection with Sample BSM.",
+	"version":"3.0.0",
```

Comment:
probably should update this to reflect V2X-Hub version or something but not a priority